Lib/test/test_httpservers.py

```diff
@@ -539,6 +539,35 @@ def test_get(self):
             finally:
                 os.chmod(self.tempdir, 0o755)
 
+    def test_range_get(self):
+        response = self.request(self.base_url + '/test')
+        self.assertEqual(response.getheader('accept-ranges'), 'bytes')
+        self.check_status_and_reason(response, HTTPStatus.OK, data=self.data)
+
```

Comment:
We need a test when both endpoints are the same: bytes=0-0 for instance and also bytes=n-n where n is the full length.

Technically, it is clear that the invalid range spec '-' is parsed as None since we have neither start nor finish so we do not that test.